src/main/scala/com/cognite/sdk/scala/v1/resources/sessions.scala

```diff
@@ -51,6 +51,14 @@ class Sessions[F[_]](val requestSession: RequestSession[F])
         uri"$baseUrl/token",
         value => value
       )
+
+  def revoke(ids: Items[CogniteInternalId]): F[Seq[CogniteInternalId]] =
+    requestSession
+      .post[Seq[CogniteInternalId], Items[CogniteInternalId], Items[CogniteInternalId]](
+        ids,
+        uri"$baseUrl/sessions/revoke",
+        value => value.items
```

Comment:
Some things that aren't clear atm
- from docs it's not clear when results are paged, we probably will need to handle that in sdk
- not clear if operation can have partial success and whether it'd mean response items length is shorter than request
- would this call still work with extra fields in response (when client has sessionsAcl:LIST)

Reply:
added a test to make sure circe ignores the extra fields if they are returned

Reply:
I think the cursor in the spec is a typo from copy paste.... because there is no `cursor` parameter, so no way to give it back to get a second page

Reply:
if my Rust isn't so rust-ed 🥁, I think the call will fail if there is more than one id

https://github.com/cognitedata/iam-api/blob/191cde8db8156a225fc0519bb0b66222632d856c/iam-sessions-api/src/v1.rs#L269

Reply:
Not sure if we should do extra validations on the output (like its lenght).... I mean, if the API returns 200 with some list, shouldn't we just return that list and let the user do what they deem needed with it?

In any case.... checked the api implementation, and it validates that all ids exist and the user is authorized to revoke them before revoking any, so partial successes seem unlikely